dev-support/Jenkinsfile

```diff
@@ -130,6 +128,7 @@ curl -L  -o personality.sh "${env.PROJECT_PERSONALITY}"
             }
           }
           environment {
+            BASEDIR = "${env.WORKSPACE}/component"
```

Comment:
I... don't understand how these are different from what we had previously?

Reply:
the captured context is different. jenkins captures things like WORKSPACE per executor. how it turns the given stages and initial environment into executors is magic AFAICT.

Reply:
so down here, because we had a `node` on the stage we get a new executor and a new WORKSPACE gets made.